package.json

```
@@ -6,19 +6,28 @@
     "@gi-types/gjs-environment": "^1.1.0",
     "@gi-types/gtk4": "^4.6.1",
     "@gi-types/gtk4-types": "^1.0.0",
-    "@rollup/plugin-commonjs": "^22.0.1",
-    "@rollup/plugin-node-resolve": "^13.3.0",
+    "@rollup/plugin-commonjs": "^25.0.4",
+    "@rollup/plugin-node-resolve": "^15.2.1",
     "events": "^3.3.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^14.0.1",
     "ltx": "git://github.com/xmppjs/ltx.git#093c7bdced57c22f7eccba3bccfc141150654395",
-    "postcss": "^8.4.14",
-    "prettier": "^2.7.1",
-    "rollup": "^2.76.0",
+    "postcss": "^8.4.28",
+    "prettier": "^3.0.2",
+    "rollup": "^3.28.1",
     "rollup-plugin-ignore": "^1.0.10",
     "rollup-plugin-node-polyfills": "^0.2.1",
-    "rome": "^12.1.1"
+    "rome": "^12.1.3"
   },
   "type": "module",
   "scripts": {
-    "postinstall": "rollup -c rollup.config.js"
+    "postinstall": "rollup -c rollup.config.js",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.{js,cjs,mjs}": "rome format --write",
+    "*.{css,json,md}": "prettier --write",
+    "*.rs": "flatpak run --command=/usr/lib/sdk/rust-stable/bin/rustfmt --file-forwarding org.gnome.Sdk//45beta @@ ",
```

```suggestion
    "*.rs": "flatpak run --command=/usr/lib/sdk/rust-stable/bin/rustfmt --edtion 2021 --file-forwarding org.gnome.Sdk//45beta @@ ",
```

Seems like that is necessary